jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java

```diff
@@ -470,7 +470,29 @@ public static boolean isBlank(String str)
         // only whitespace
         return true;
     }
-    
+
+    /**
+     * <p>Checks if a String is empty ("") or null.</p>
+     *
+     * <pre>
+     *   isEmpty(null)   == true
+     *   isEmpty("")     == true
+     *   isEmpty("\r\n") == false
+     *   isEmpty("\t")   == false
+     *   isEmpty("   ")  == false
+     *   isEmpty("a")    == false
+     *   isEmpty(".")    == false
+     *   isEmpty(";\n")  == false
+     * </pre>
+     *
+     * @param str
+     *            the string to test.
+     * @return true if string is null or only whitespace characters, false if non-whitespace characters encountered.
+     */
+    public static boolean isEmpty(String str){
```

Comment:
probably `isNullOrEmpty` is a better name... other than that LGTM

Reply:
I like `isEmpty()` better.

Reply:
I can live with that.....  but I will NEVER accept the curly in that position :)

Reply:
formatting fixed